coral/src/app/features/browse-topics/components/topic-table/TopicTable.tsx

```diff
@@ -36,19 +36,19 @@ function TopicTable(props: TopicListProps) {
               align={"left"}
               className={classes.topicTableRowHead}
             >
-              {dot.topicName}
+              <a href={`/topicOverview?topicname=${dot.topicName}`}>
+                {dot.topicName}
```

Comment:
General curiosity: is there a reason do we name the argument to the `map` callback `dot`? A convention?

Reply:
Thanks for raising this! I copied that from the DS docu and didn't change the variable 🙈  Will update with a better naming!